packages/nimble-components/package.json

```diff
@@ -44,6 +40,53 @@
     "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
     "test": "npm run test-chrome:verbose && npm run test-firefox:verbose"
   },
+  "wireit": {
+    "build": {
+      "dependencies": ["../nimble-tokens:build", "bundle-components", "generate-scss", "build-storybook"]
+    },
+    "generate-icons": {
+      "command": "node build/generate-icons/dist/index.js",
+      "dependencies": ["generate-icons:bundle"],
+      "files": ["build/generate-icons/dist/index.js", "../nimble-tokens/dist"],
+      "output": ["src/icons/*.ts"]
+    },
+    "generate-icons:bundle": {
+      "command": "rollup --bundleConfigAsCjs --config build/generate-icons/rollup.config.js",
+      "files": ["build/generate-icons/source/index.js", "build/generate-icons/rollup.config.js"],
+      "output": ["build/generate-icons/dist/index.js"]
+    },
+    "build-components": {
+      "command": "tsc --build --pretty ./tsconfig.json",
+      "clean": "if-file-deleted",
```

Comment:
Do we need `"clean": "if-file-deleted",`for `tsc` commands in other projects? Should we enable incremental build support for them too?